### PR TITLE
IR: cleanup

### LINKED ISSUE #15831: Broken: Iranian channels
### Broken Link

https://iptv-org.github.io/iptv/countries/ir.m3u

### What happened to the stream?

Not loading

### Notes (optional)

Almost all of the 136 channels in Iranian channels are broken im not asking for them to be fixed or anything just wanted to inform u about this and ty for making this :)

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Dum4G**
Resolves https://github.com/iptv-org/iptv/issues/15831

I was zero-tolerant to Not 24/7 tag, not pushing this PR to be immediately approved

### CODE REVIEW

File: streams/uk.m3u [hunk 1]
```diff
@@ -94,10 +94,6 @@ https://live.islamchannel.tv/live7/islamtv_english/bitrate1.isml/live.m3u8
 https://live-islamtv-urdu.simplestreamcdn.com/live7/islamtv_urdu/bitrate1.isml/live.m3u8
 #EXTINF:-1 tvg-id="JewelleryMaker.uk",Jewelery Maker (1080p)
 https://lo2-1.gemporia.com/abrjewellerymaker/smil:livestream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="",JimJam Россия (576p) [Not 24/7]
-http://188.40.68.167/russia/jimjam/playlist.m3u8
-#EXTINF:-1 tvg-id="KalemehTV.uk",Kalemeh TV (576p) [Not 24/7]
-http://51.210.199.37/hls/stream.m3u8
```
Comment (CasperMcFadden95) [inline streams/uk.m3u:100]: https://5e11be0b2c93a.streamlock.net/live/ngrp:kalemeh_all/playlist.m3u8

File: streams/ir.m3u [hunk 2]
```diff
@@ -41,78 +31,20 @@ https://damac-tv.icdndhcp.com/hls/stream.m3u8
...
-#EXTINF:-1 tvg-id="HodHodFarsiTV.ir",HodHod Farsi TV (720p)
-https://hls.hodhod.live/hls/stream.m3u8
 #EXTINF:-1 tvg-id="",ICC TV (1080p)
 https://icchls.wns.live/hls/stream.m3u8
 #EXTINF:-1 tvg-id="ICnet1.ca",ICnet 1 (720p) [Not 24/7]
 https://60ba7ef02e687.streamlock.net/ICNET/ngrp:my.stream_all/playlist.m3u8
-#EXTINF:-1 tvg-id="iFilm2.ir",iFilm 2 (480p)
-https://live1.presstv.ir/live/ifilm2.m3u8
-#EXTINF:-1 tvg-id="iFilmArabic.ir",iFilm Arabic (720p) [Not 24/7]
-https://cdnlive.presstv.ir/cdnlive/smil:ifilmarcdn.smil/playlist.m3u8
```
Comment (CasperMcFadden95) [inline streams/ir.m3u:107]: https://cdnlive.presstv.ir/ifilmlive/smil:ifilmar.smil/playlist.m3u8

File: streams/ir.m3u [hunk 3]
```diff
@@ -41,78 +31,20 @@ https://damac-tv.icdndhcp.com/hls/stream.m3u8
...
-#EXTINF:-1 tvg-id="HodHodArabicTV.ir",HodHod Arabic TV (540p)
-https://nl2.livekadeh.com/hls2/hodhodarabic.m3u8
-#EXTINF:-1 tvg-id="HodHodFarsiTV.ir",HodHod Farsi TV (720p)
-https://hls.hodhod.live/hls/stream.m3u8
 #EXTINF:-1 tvg-id="",ICC TV (1080p)
 https://icchls.wns.live/hls/stream.m3u8
 #EXTINF:-1 tvg-id="ICnet1.ca",ICnet 1 (720p) [Not 24/7]
 https://60ba7ef02e687.streamlock.net/ICNET/ngrp:my.stream_all/playlist.m3u8
-#EXTINF:-1 tvg-id="iFilm2.ir",iFilm 2 (480p)
-https://live1.presstv.ir/live/ifilm2.m3u8
```
Comment (CasperMcFadden95) [inline streams/ir.m3u:105]: https://cdnlive.presstv.ir/ifilmlive/smil:ifilm2.smil/playlist.m3u8

File: streams/ir.m3u [hunk 4]
```diff
@@ -41,78 +31,20 @@ https://damac-tv.icdndhcp.com/hls/stream.m3u8
...
-#EXTINF:-1 tvg-id="iFilmArabic.ir",iFilm Arabic (720p) [Not 24/7]
-https://cdnlive.presstv.ir/cdnlive/smil:ifilmarcdn.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="iFilmArabic.ir",iFilm Arabic (720p) [Not 24/7]
-https://live1.presstv.ir/live/ifilmar.m3u8
-#EXTINF:-1 tvg-id="iFilmEnglish.ir",iFilm English (480p)
-https://live1.presstv.ir/live/ifilmen.m3u8
```
Comment (CasperMcFadden95) [inline streams/ir.m3u:111]: https://cdnlive.presstv.ir/ifilmlive/smil:ifilmen.smil/playlist.m3u8

File: streams/ir.m3u [hunk 5]
```diff
@@ -41,78 +31,20 @@ https://damac-tv.icdndhcp.com/hls/stream.m3u8
...
-#EXTINF:-1 tvg-id="iFilmPersian.ir",iFilm Persian (540p)
-https://live.aionet.ir/hls/ifilm/ifilm.m3u8
```
Comment (CasperMcFadden95) [inline streams/ir.m3u:113]: https://cdnlive.presstv.ir/ifilmlive/smil:ifilmfa.smil/playlist.m3u8

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by sguinetti — APPROVED** [review]

**Pull request merged**